bigbluebutton-html5/private/config/settings.yml

```
@@ -957,7 +957,38 @@ public:
     log: true
+    # Client-side WebRTC stats logging. If enabled, the client will log periodic
+    # WebRTC stats information to the configured logging transports.
+    # Disabled by default. This log is verbose and may incur in additional network
+    # usage if external logging targets are configured.
+    # The effective stats logging frequency is determined by the ratio of
+    # stats.interval to stats.probesd (interval / probes).
+    logMediaStats:
+      enabled: false
+      # Defines additional WebRTC stats to extract from getStats(), beyond the default ones.
+      # Applied only when logMediaStats.enabled is true.
+      # 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)